docs/vspecID.md

```diff
@@ -0,0 +1,103 @@
+# vspecID - vspec static UID generator and validator
+
+The vspecID.py script is used to generate and validate static UIDs for all nodes in the tree.
+They will be used as unique identifiers to transmit data between nodes. The static UIDs are
+implemented to replace long strings like `Vehicle.Body.Lights.DirectionIndicator.Right.IsSignaling`
+with a 4-byte identifier.
+
+ToDo what kind of data does each byte represent using latex
+
+## General usage
+
+```bash
+usage: vspecID.py [-h] [-I dir] [-e EXTENDED_ATTRIBUTES] [-s] [--abort-on-unknown-attribute] [--abort-on-name-style] [--format format] [--uuid] [--no-expand] [-o overlays]
```

Comment:
Two thoughts:

- Shall we use `vspec2...` naming pattern like for other tools, i.e. `vspec2id`?
- Here we can remove a lot of arguments that does not make sense to use, like `--format format` and `--json-pretty`